### PR TITLE
Refactored and expanded dogs functionality

### PR DESCRIPTION
**Pull request by Asterids**
- Moved 'phrases' into directory
- Created additional supported dogs and refactored
- Removed unnecessary check from regex in index.js

### CODE REVIEW

File: operations/phrases/dogs.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const axios = require('axios');
+
+module.exports = dogPhrases = async (args, message) => {
+  const supportedDogs = [
+    'dachsund', 'pug', 'schipperke', 'shiba', 'whippet'
```
Comment (amydegenaro) [inline operations/phrases/dogs.js:5]: as one who prefers big fluffy dogs, can I pls request the addition of samoyed :D

File: operations/phrases/dogs.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  let dogFound = null;
+  supportedDogs.forEach((dog) => {
+    if (args.includes(dog)) {
+      dogFound = dog;
+    }
+  });
+
+  if (dogFound) {
+    const dog = await axios.get(`https://dog.ceo/api/breed/${dogFound}/images/random`);
+    message.channel.send(`${dogFound} acquired`, { files: [ dog.data.message ] });
```
Comment (mincer-ray) [inline operations/phrases/dogs.js:17]: would you be willing to write some error handling for this? like check if `dog.data` axios response is 200 and exists and if not send some kind of error to chat instead.

**Comment by mincer-ray** [comment]
make sure to pass linting. dont worry about the errors

File: operations/phrases/dogs.js (hunk 1 above)
Comment (Asterids) [reply operations/phrases/dogs.js:5]: Yes, I'll add it! :)

File: operations/phrases/dogs.js (hunk 2 above)
Comment (Asterids) [reply operations/phrases/dogs.js:17]: Yes! Error handling is important

**Pull request merged**